tests/console/dracut_enhanced.pm

```diff
@@ -30,11 +30,16 @@ use version_utils 'is_sle';
 sub run {
     select_console 'root-console';
     my $dracut_test = get_var('DRACUT_TEST');
+    # dracut on SLE15SP2 was updated to 049
+    my $dracut_version = "dracut-patches.tar.gz";
+    if (is_sle('>=15-SP2')) {
+            $dracut_version = "dracut-patches-SLE15SP2.tar.gz";
```

Comment:
Good work @emiura, lgtm! I think that Travis is failing here for the space, but I think you can run `./tools/tidy` and it will be ok :)

Reply:
@punkioudi My tidy does nothing locally. My new vm on conan is a bit weird.